src/components/Info/Info.jsx

```diff
@@ -15,36 +16,54 @@ function Info(props) {
         setActiveTab(2)
     }
 
+    // TO-DO: refactor this code to improve how we're handling text
     return <Modal>
-        <ModalContent>
             <TabBar fixed>
                 <Tab onClick={handleTabSwitch1} selected={activeTab === 1} dataTest="dhis2-uicore-tab">{i18n.t("User Guide")}</Tab>
                 <Tab onClick={handleTabSwitch2} selected={activeTab === 2} dataTest="dhis2-uicore-tab">{i18n.t("Additional Features")}</Tab>
             </TabBar>
+            <ModalContent className={styles.info}>
             {activeTab === 1 ? <div>
-                <h5>How to reate new catchment areas</h5>
-                <p>Click INSERT BUTTON</p>
-                <p>Select the country you want to target</p>
-                <p>Name the catchment areas</p>
-                <p>Choose the admin level where your facilities are located</p>
-                <p>Choose which groups of facilities to include</p>
-                <p>You can use your catchment areas to create thematic maps and other views in the maps module of DHIS2</p>
-                <p>1</p>
+                <ModalTitle>{i18n.t("How to create new catchment areas")}</ModalTitle>
+                <p className={styles.align}>{i18n.t("Click")}<ButtonItem primary={true} buttonText={i18n.t("Create")}/></p>
+                <p>{i18n.t("Select the country you want to target.")}</p>
+                <p>{i18n.t("Name the catchment areas.")}</p>
+                <p>{i18n.t("Choose the admin level where your facilities are located.")}</p>
+                <p>{i18n.t("Choose which groups of facilities to include.")}</p>
+                <p>{i18n.t("You can use your catchment areas to create thematic maps and other views in the maps module of DHIS2.")}</p>
+                <Divider/>
+                <ModalTitle>{i18n.t("Understanding the statuses")}</ModalTitle>
+                <p>{i18n.t("Pending")}: {i18n.t("your catchment areas are being created.")}</p>
+                <p>{i18n.t("Ready")}: {i18n.t("your catchment areas have been created and ready to be published to DHIS2.")}</p>
+                <p>{i18n.t("Publishing")}: {i18n.t("your catchment areas is in the process of being published.")}</p>
+                <p>{i18n.t("Published")}: {i18n.t("your catchment areas have been published to DHIS2.")}</p>
```

Comment:
I think it would be good if `Pending`, `Ready`, etc. were bolded or otherwise easy to differentiate from the description text.